MinecraftClient/Protocol/Handlers/Packet/s2c/DeclareCommands.cs

```diff
@@ -92,6 +94,16 @@ public static void Read(DataTypes dataTypes, Queue<byte> packetData, int protoco
                             42 => new ParserResourceOrTag(dataTypes, packetData),
                             43 => new ParserResource(dataTypes, packetData),
                             44 => new ParserResource(dataTypes, packetData),
+                            50 => protocolVersion == Protocol18Handler.MC_1_19_4_Version ?
+                              new ParserForgeEnum(dataTypes, packetData) :
+                              new ParserEmpty(dataTypes, packetData),
+                            51 => (protocolVersion >= Protocol18Handler.MC_1_20_Version &&
+                                   protocolVersion <= Protocol18Handler.MC_1_20_2_Version) ? // 1.20 - 1.20.2
+                              new ParserForgeEnum(dataTypes, packetData) :
+                              new ParserEmpty(dataTypes, packetData),
+                            52 => protocolVersion > Protocol18Handler.MC_1_20_2_Version ? // 1.20.2 +
+                              new ParserForgeEnum(dataTypes, packetData) :
+                              new ParserEmpty(dataTypes, packetData),
```

Comment:
There will be a new switch block for 1.20.3+, so would you mind removing the 1.20.3+ forge parser in this PR and add it back once the 1.20.3/4 PR is merged?

Reply:
Okay, it's done.

Reply:
> Okay, it's done.

I just merged the 1.20.3 branch, please pull from master in to your branch and then re-add anything that needs to be added.